automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/library/DataLibraryPage.scala

```diff
@@ -16,6 +16,10 @@ class DataLibraryPage(implicit webDriver: WebDriver) extends BaseFireCloudPage
 
   private val LibraryTable = Table("library-table")
   private val searchField = SearchField("library-search-input")
+  private val rpModalLink = Link("show-research-purpose-modal")
```

Comment:
I was thinking about making a new class to represent the modal separately, but it's so simple I decided not to for now

Reply:
Sometime soon, I'm going to make Modal a Component a la Collapse. For now this is fine.

Reply:
we already have an OKCancelModal abstract class. org.broadinstitute.dsde.firecloud.page.Modals.scala